src/bot.ts

```diff
@@ -56,15 +58,14 @@ class Bot {
 			}
 		});
 
-		this.interval = setInterval(async () => {
-			const count = await this.meetSession.participants();
-			const status = `${count} ${
-				count === 1 ? 'person is' : 'people are'
-			} in this call`;
-			if (this.zulipClient.profile.status !== status) {
-				await this.zulipClient.setStatus(status);
-			}
-		}, bConfig.statusInterval);
+		// Periodically check participants
+		this.interval = setInterval(
+			this.checkParticipants.bind(this),
+			bConfig.statusInterval,
+		);
+
+		// Check participants right away
+		await this.checkParticipants();
```

Comment:
before the bot would wait the interval (1 minute) before checking the participants.